tests/api/test_variant.py

```diff
@@ -150,6 +150,94 @@ def test_create_variant(admin_api_client, product, product_type):
     assert data['weight']['value'] == weight
 
 
+def test_create_product_variant_not_all_attributes(
+        admin_api_client, product, product_type, color_attribute):
+    query = """
+            mutation createVariant (
+                $productId: ID!,
+                $sku: String!,
```

Comment:
Does this query needs all of the fields listed here? It's kinda long for checking only if attributes are present.